README.md

```diff
@@ -165,7 +165,7 @@ cd deephaven-core
 python3 -m venv ~/my-dh-venv
 source ~/my-dh-venv/bin/activate
 ./gradlew py-server:assemble
-pip install "py/server/build/wheel/deephaven_core-<version>-py3-non-any.whl[autocomplete]
+pip install py/server/build/wheel/deephaven_core-<version>-py3-non-any.whl[autocomplete]
```

Comment:
There should be quotes around this command. Without the quotes, the command fails. 
```
(.venv) ➜  main git:(main) pip install /home/bender/dev/deephaven/iris-oss/core/main/py/server/build/wheel/deephaven_core-0.39.0+snapshot-py3-none-any.whl[autocomplete]                      
zsh: no matches found: /home/bender/dev/deephaven/iris-oss/core/main/py/server/build/wheel/deephaven_core-0.39.0+snapshot-py3-none-any.whl[autocomplete]
```

https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-extras

Reply:
Created a PR to fix: https://github.com/deephaven/deephaven-core/pull/6806